templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -529,7 +572,7 @@ contents:
         # We also rollback if it looks like we need to configure things, just in case there are any leftovers
         # from previous attempts.
         if [ -f "$extra_bridge_file" ] || [ -z "$(nmcli connection show --active br-ex 2> /dev/null)" ]; then
-          echo "Bridge br-ex is not active, restoring previous configuration before proceeding..."
+          echo "Restoring previous configuration before proceeding..."
```

Comment:
nit: found the original message useful

Reply:
I reverted it back to what it was before, sorry.